tdp/core/variables/cluster_variables.py

```diff
@@ -52,9 +52,11 @@ def initialize_cluster_variables(
             (collection_name, collection.default_vars_directory.iterdir())
             for collection_name, collection in collections.items()
         ]
-        if override_folder:
-            override_folder = Path(override_folder)
-            collections_and_overrides.append(("overrides", override_folder.iterdir()))
+
+        for i, override_folder in enumerate(override_folders):
+            collections_and_overrides.append(
```

Comment:
We accept folder as `str` and `Path` for `override_folder`, if you have a `str` the `.iterdir()` will fail, you must keep the `override_folder = Path(override_folder)`.